fixture-monkey-kotlin/src/main/kotlin/com/navercorp/fixturemonkey/kotlin/KotlinPlugin.kt

```diff
@@ -38,14 +41,22 @@ import com.navercorp.fixturemonkey.kotlin.matcher.Matchers.TRIPLE_TYPE_MATCHER
 import com.navercorp.fixturemonkey.kotlin.property.KotlinPropertyGenerator
 import com.navercorp.fixturemonkey.kotlin.type.actualType
 import com.navercorp.fixturemonkey.kotlin.type.cachedKotlin
+import java.lang.reflect.Modifier
 import org.apiguardian.api.API
 import org.apiguardian.api.API.Status.MAINTAINED
-import java.lang.reflect.Modifier
 
 @API(since = "0.4.0", status = MAINTAINED)
-class KotlinPlugin : Plugin {
+class KotlinPlugin(
+    private val kotlinIntrospector: ArbitraryIntrospector = PrimaryConstructorArbitraryIntrospector.INSTANCE,
+    private val javaIntrospector: ArbitraryIntrospector = BeanArbitraryIntrospector.INSTANCE,
+) : Plugin {
     override fun accept(optionsBuilder: FixtureMonkeyOptionsBuilder) {
-        optionsBuilder.objectIntrospector { PrimaryConstructorArbitraryIntrospector.INSTANCE }
+        optionsBuilder.objectIntrospector {
+            KotlinAndJavaCompositeArbitraryIntrospector(
+                kotlinIntrospector = kotlinIntrospector,
+                javaIntrospector = javaIntrospector
+            )
+        }
```

Comment:
I think that `javaIntrospector` is not concern of `KotlinPlugin`.
It may cause a breaking change in the near future.

It is better to use `objectInrospector` option to apply new `KotlinAndJavaCompositeArbitraryIntrospector` with desired `javaIntrospector`

Reply:
I also feel that it's excessive for the `KotlinPlugin` to be setting up the `JavaIntrospector`. However, it seems good for objects to be smoothly created with the `KotlinPlugin` alone(dont use other api), regardless of whether Java reference types are used internally.

If a design is planned for a future version (1.1.x) that allows for such behavior, 

then in the current PR, I plan to exclude changes to the `KotlinPlugin` and only keep the implementation of `KotlinAndJavaCompositeArbitraryIntrospector`.

Reply:
@jinia91 
> then in the current PR, I plan to exclude changes to the KotlinPlugin and only keep the implementation of KotlinAndJavaCompositeArbitraryIntrospector.

👍  It is good idea.